packages/data/src/components/use-select/test/index.js

```diff
@@ -203,4 +203,79 @@ describe( 'useSelect', () => {
 			}
 		);
 	} );
+
+	it( 'uses memoized selector if dependent stores do not change', async () => {
+		const storeConfig = {
+			actions: {
+				increment: () => ( { type: 'INCREMENT' } ),
+			},
+			reducer: ( state, action ) => {
+				if ( ! state ) {
+					return { counter: 0 };
+				}
+				if ( action?.type === 'INCREMENT' ) {
+					return { counter: state.counter + 1 };
+				}
+				return state;
+			},
+			selectors: {
+				getCounter: ( state ) => state.counter,
+			},
+		};
+		registry.registerStore( 'store-1', storeConfig );
+		registry.registerStore( 'store-2', storeConfig );
+
+		const selectSpyFoo = jest
+			.fn()
+			.mockImplementation( ( { getCounter } ) => getCounter() );
```

Comment:
Nitpick: We can just write it like

```suggestion
		const selectSpyFoo = jest.fn( ( { getCounter } ) => getCounter() );
```

Reply:
Isn't it how Prettier formats it? I would personally leave all formatting choices to tools.

Reply:
I think @kevin940726  meant omitting the `.mockImplementation` part